live-css-editor/src/App.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
you can also have your imports from react in one line with `import React, {useState} from 'react';`